doc/source/coding-style/index.rst

```diff
@@ -0,0 +1,29 @@
+Coding Style
+############
+
+Coding Style refers to the different rules defined in a software project which
+need to be followed when writing source code. These guidelines ensure that all
+the source code is going to look the same across the different files of the
+project.
+
+The PyAnsys ecosystem is composed by multiple projects. To unify the coding
+style among all projects, this coding style guideline was devised. 
```

Comment:
```suggestion
Because the PyAnsys ecosystem consists of many projects, coding style rules
are critical to ensuring that the source code will look the same across different
projects.
```